presto-hive/src/main/java/io/prestosql/plugin/hive/s3/HiveS3Config.java

```diff
@@ -66,6 +66,15 @@
     private boolean skipGlacierObjects;
     private boolean requesterPaysEnabled;
 
+    /*
+     * Sentinel unicode comment character (http://www.unicode.org/faq/private_use.html#nonchar_codes).
+     * It is expected that \uFDD0 sentinel comment character is not the first character in any row of user's CSV S3 object.
+     * The rows starting with \uFDD0 will be skipped by S3Select and will not be a part of the result set or aggregations.
+     * To process CSV objects that may contain \uFDD0 as first row character please disable S3SelectPushdown.
+     * TODO: Remove this proxy logic when S3Select API supports disabling of row level comments.
+     */
+    private String s3CsvComments = "\uFDD0";
```

Comment:
Let's create a new config class `HiveS3SelectConfig` for this property. Name the config property `hive.s3select-pushdown.csv-comments` to match the other S3Select comments.

Later, we can move the existing S3Select configs from `HiveConfig` to the new `HiveS3SelectConfig`.